src/app/locator/qgsgotolocatorfilter.cpp

```diff
@@ -40,15 +42,23 @@ void QgsGotoLocatorFilter::fetchResults( const QString &string, const QgsLocator
 
   const QgsCoordinateReferenceSystem currentCrs = QgisApp::instance()->mapCanvas()->mapSettings().destinationCrs();
   const QgsCoordinateReferenceSystem wgs84Crs( QStringLiteral( "EPSG:4326" ) );
+  QgsCoordinateReferenceSystem extraCrs( QStringLiteral( "EPSG:4326" ) );
+
+  QgsSettings settings;
+  QString extraCrsAuthId = extraCrs.authid();
+  if ( settings.value( QStringLiteral( "locator_filters/go_to_coordinate/extra_crs" ), QStringLiteral( "EPSG:4326" ), QgsSettings::App ) != QStringLiteral( "EPSG:4326" ) )
```

Comment:
can you declare the setting according to the new guide lines?
https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html#settings